addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -254,8 +254,10 @@ def get_vat_country(vat):
             'pdf': pdf,
             'pdf_name': pdf_name,
         }
-        content = self.env.ref('l10n_it_edi.account_invoice_it_FatturaPA_export').render(template_values)
-        return content
+        xml_content = b"<?xml version='1.0' encoding='UTF-8'?>"
```

Comment:
maybe we could add the xml header in the generic module?

Reply:
As discussed live, this would create some cluter in the code since we don't return xml_content directly but rather a dictionary with different values. It would require some additional nasty computation.